build.gradle

```diff
@@ -817,16 +828,21 @@ publishing {
 	}
 }
 
-
 void mapJar(File output, File input, File mappings, File libraries, String from, String to) {
+	mapJar(output, input, mappings, libraries, from, to, { })
+}
+
+void mapJar(File output, File input, File mappings, File libraries, String from, String to, Action<TinyRemapper.Builder> action) {
```

Comment:
I think you can just do:

```groovy
void mapJar(File output, File input, File mappings, File libraries, String from, String to, Action<TinyRemapper.Builder> action = { }) {
```

Reply:
hmm, groovy has default args? let me try

Reply:
It works perfectly! wrapped the line for the action arg because otherwise the line feels too long

Reply:
Yeah, groovy has a lot of little tricks like this.